website/assets/js/pages/customers/new-license.page.js

```diff
@@ -40,7 +41,14 @@ parasails.registerPage('new-license', {
     //…
   },
   mounted: async function() {
-    //…
+
+    // If this users firstName and lastName are set to be part of the user's email address, they signed up through the Fleet Sandbox signup, and we'll need some additional information to complete this order.
```

Comment:
Could we store [signupReason](https://github.com/fleetdm/fleet/blob/5693571b782c513651bc9540d650728fb449c158/website/api/controllers/entrance/signup.js#L59-L63) on the User and check that instead?

i.e. `this.me.signupReason ===`

I think that would be more explicit, and less likely to be forgotten about and surprise us later.

Reply:
@mikermcneil I think that approach makes more sense. 

For existing fleetdm.com accounts we would have to default this value to the sandbox signupReason ('Try Fleet Sandbox`). 
This could lead to a user who signed up through the license dispenser being required to reenter their information before they purchase a license.

Another option would be to check if `this.me.organization === ''`. `organization` is a required input on the license dispenser sign-up page, but if they signed up through the sandbox registration page, it will be set to a blank string.

Reply:
Could we use those tactics to migrate the existing data?  And then start storing signupReason?